server/src/services/FroshServices.js

```diff
@@ -56,6 +56,72 @@ const FroshServices = {
     await froshGroup.save();
     await newUserSubscription.add(froshRecord);
   },
+
+  // async updateFroshDataField(email) {
+  //   let errors = [];
+  //   FroshModel.findOne({ email }).then((frosh) => {
+  //     if (!frosh) {
+  //       console.log("error");
+  //       errors.push('The frosh doesn\'t exist');
+  //     } else {
+  //       if (errors.length === 0) {
+  //         /*Update the field */
+  //         frosh[field] = value;
+  //         // frosh["lastUpdatedAcct"] = new Date();
+  //         // frosh["lastUpdatedFields"].push(field);
+  //         frosh.save();
+  //         console.log("frosh edited");
+  //       }
+	// 	  }
+	//   });
+  //   return errors;
+  // },
+
+  // async emailRegExpFindFrosh(froshEmail) {
+  //   return (Frosh.findOne({
+  //     email: new RegExp(`^${froshEmail}$`, 'i')
+  //   }));
+  // },
+
+  // async resetPassTokenFindFrosh(token) {
+  //   return (Frosh.findOne({
+  //     resetPasswordToken: token,
+  //   }));
+  // },
+  
+  async initFroshGroups(groups) {
```

Comment:
Please add a [jsdoc comment](https://jsdoc.app/about-getting-started.html) to document this function - all you need is a one sentence description, and then params and a return type